pkg/sender/types.go

```diff
@@ -46,4 +46,8 @@ type Sender interface {
 	// StopSending will stop any long running maintenance threads in the sender plugin.
 	// Often this function does nothing.
 	StopSending(ctx context.Context)
+	//
+	Config() interface{}
+	Name() string
+	Plugin() string
```

Comment:
Why doesn't Filter interface also have these three methods?

Reply:
i didn't have the patience to implement this for filters yet and i only really needed visibility into sender and receivers anyway, but you are right, will add the missing apis shortly